packages/stencil/src/components/datacom-input/datacom-input.css

```diff
@@ -51,6 +51,7 @@
   caret-color: var(--dc-primary-colour);
   text-overflow: ellipsis;
   width: 100%;
+  background-color: white;
```

Comment:
You think there's merit to making that white color value into a CSS variable?

Something like:
```
--dc-white: #ffffff
```

I recognize that there is a "--dc-background-colour", but could be useful to have a more generic white color variable for use in other places than a background.

Reply:
I'll just use --dc-background-colour